server/graphql.ts

```diff
@@ -242,6 +242,12 @@ function resolver(registration: Registration): IResolver {
                     return null;
                 }
 
+                // If warnOnDuplicates is not set for this tag, set it to true (most restrictive option)
+                if (!tagDetails.warnOnDuplicates) {
```

Comment:
Sorry, I got this wrong. The condition should be `if (tagDetails.warnOnDuplicates !== false)` to catch `true`, `undefined`, and `null`, but not an explicit `false`.